mysql-test/suite/galera/t/galera_ist_mariabackup_verify_ca.test

```diff
@@ -14,6 +14,7 @@
 CREATE TABLE t1 (f1 INT PRIMARY KEY);
 
 --connection node_2
+let $innodb_log_file_buffering_orig=`SELECT @@innodb_log_file_buffering`;
```

Comment:
Please do not use `let` but
```sql
set @log_file_buffering=@@innodb_log_file_buffering;
```

Reply:
This does not work, mysqltest: At line 69: query 'SET GLOBAL innodb_log_file_buffering = @log_file_buffering' failed: ER_WRONG_VALUE_FOR_VAR (1231): Variable 'innodb_log_file_buffering' can't be set to the value of 'NULL'

Reply:
I think that a cleaner fix would be to ensure that the size of the dummy log file is 4096 bytes, which is the maximum physical block size that we support for `O_DIRECT` access:
```diff
diff --git a/extra/mariabackup/xtrabackup.cc b/extra/mariabackup/xtrabackup.cc
index c53ff0792c3..92e7e661422 100644
--- a/extra/mariabackup/xtrabackup.cc
+++ b/extra/mariabackup/xtrabackup.cc
@@ -2577,7 +2577,7 @@ static bool innodb_init_param()
 	return true;
 }
 
-static byte log_hdr_buf[log_t::START_OFFSET + SIZE_OF_FILE_CHECKPOINT];
+static byte log_hdr_buf[4096];
 
 /** Initialize an InnoDB log file header in log_hdr_buf[] */
 static void log_hdr_init()
```

Reply:
If you run the affected tests with `--innodb-log-file-buffering`, then no attempt to open log files with `O_DIRECT` will be made an MTR's internal check of the test case cannot possibly fail.